fuse_examples/tutorials/hello_world/hello_world.ipynb

```diff
@@ -47,9 +47,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!git clone https://github.com/IBM/fuse-med-ml.git\n",
-    "%cd fuse-med-ml\n",
-    "!pip install -e ."
+    "install_fuse = False # change to 'True' to clone and install fuse-med-ml. \n",
```

Comment:
@mosheraboh 
Yesterday I've opened a pull request on testbook's repo adding a 'skip' feature 🙂 
When they'll approve it I'll be able to clean it.